src/display/api.js

```diff
@@ -152,6 +152,9 @@ function setPDFNetworkStreamFactory(pdfNetworkStreamFactory) {
  *   parsed font data from the worker-thread. This may be useful for debugging
  *   purposes (and backwards compatibility), but note that it will lead to
  *   increased memory usage. The default value is `false`.
+ * @property {HTMLDocument} [ownerDocument] - Specify an explicit document
+ *   context to create elements for and to load resouces into, such as
```

Comment:
Should `for` perhaps be replaced by `with` above?

Also, please fix the spelling of `resouces` -> `resources` here.

Reply:
derp. Sorry about that.